src/tests/rooms.test.js

```diff
@@ -0,0 +1,252 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import { describe, it } from 'mocha';
+import app from '../index';
+
+chai.use(chaiHttp);
+const prefix = '/api/v1/accommodations';
+
+describe('', () => {
+  it('should sign in a user first', (done) => {
+    chai
+      .request(app)
+      .post('/api/v1/auth/signin')
+      .send({
+        email: 'octopusbn@gmail.com',
+        password: 'password'
+      })
+      .end((err, res) => {
+        expect(res.body.status).to.be.equal(200);
+        expect(res);
+        done();
+      });
+  });
+  it('should return an error message if accommodation id is not found', (done) => {
+    chai
+      .request(app)
+      .post(`${prefix}/room`)
+      .send(
+        {
+          accommodationsID: '8a2c44a8-d3a5-451b-be73-26ded3b9e75c',
+          roomNumber: 'H145',
+          cost: 100,
+          currency: 'hello',
+          type: 'Single'
+        }
```

Comment:
@blaise82 Since we have a mock folder to keep track of these mocks:
  ``` {
          accommodationsID: '8a2c44a8-d3a5-451b-be73-26ded3b9e75c',
          roomNumber: 'H145',
          cost: 100,
          currency: 'hello',
          type: 'Single'
        }
```
I think it would be better to create a mock file for all data you use to test your feature